front/mailing/20240308-weekly-update.ts

```diff
@@ -0,0 +1,157 @@
+      • All custom assistants using Claude 2.1 have been automatically upgraded to Claude 3 Opus.<br/>
+      🔗 <a href="https://www.anthropic.com/news/claude-3-family">https://www.anthropic.com/news/claude-3-family</a>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>📊 Introducing Table Queries</b><br/>
+      • Connect Notion databases, Google Sheets, and CSV files to Dust for powerful data analysis.<br/>
+      • Ask questions like "Show me top customers by revenue" and get back instant charts and insights.<br/>
+      • Enables everyone to make data-informed decisions without needing SQL or analytics expertise.<br/>
+      • Tables in Dust are automatically kept in sync with their source Google Sheets and Notion databases.<br/>
+      🔗 <a href="https://blog.dust.tt/dusts-for-quantitative-analysis-with-llms/">https://blog.dust.tt/dusts-for-quantitative-analysis-with-llms/</a>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>📘 Confluence Connection</b><br/>
+      • Connect your Confluence instance to Dust to sync global spaces and pages.<br/>
+      • Simply add Confluence as a data source and let your assistants tap into that collective wisdom.<br/>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>💬 Intercom Connection</b><br/>
+      • Connect Intercom to sync Help Center articles and customer conversations to Dust.<br/>
+      • Choose which Teams to sync conversations from and control access to customer data.<br/>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>🔒 Okta Single Sign-On</b><br/>
+      • Enterprise customers can ask to enable Okta SSO for an even more seamless and secure authentication experience. Simplify user management by leveraging your existing Okta setup.<br/>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>🤖 Summon Assistants in Slack</b><br/>
+      • Interact with any Dust assistant right from Slack using the @dust ~assistantname or @dust +assistantname syntax. Bring the power of Dust to the tools you use every day.<br/>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>📘 Quick Start Guide</b><br/>
+      • We've added a handy walkthrough on first login to explain Dust fundamentals and help new users get oriented. Be sure to check it out!<br/>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      <b>➕ And more...</b><br/>
+      • Refreshed assistant details page and builder UX.<br/>
+      • Experimental instruction suggestions and website crawling settings.<br/>
+      • New blog posts: <a href="https://blog.dust.tt/pennylane-dust-customer-support-journey/">How Eléonore improved the efficiency of Pennylane’s Care team thanks to Dust</a> and <a href="https://blog.dust.tt/why-dust/">Why Dust</a>.<br/>
+    </p>
+
+    <p style="max-width: 500px; padding-top: 20px;">
+      Happy building,
+    </p>
+    <p>
+      The Dust Team
+    </p>
```

Comment:
Proposal:

PS: we're still working on getting a proper emailing system. If you want to unsubscribe or have any question about Dust, just reply to the email.

Reply:
@gabhubert @Yutcam wdyt?